decoders/0310-ssh_decoders.xml

```diff
@@ -63,8 +63,8 @@
 <decoder name="sshd-success">
   <parent>sshd</parent>
   <prematch>^Accepted</prematch>
-  <regex offset="after_prematch">^ \S+ for (\S+) from (\S+) port </regex>
-  <order>user, srcip</order>
+  <regex offset="after_prematch">^ \S+ for (\S+) from (\S+) port (\S+) (\S+)</regex>
```

Comment:
This change is not in the scope of this PR. In addition, we have to review the rest of decoders, for example, `sshd-failed` and `ssh-invfailed` have the same lack of decoded fields. In addition, the `sshd-failed` decoder stores the port in `srcport` instead of `port`.

Let's open a new issue to treat this decoders.